portfolio/src/main/webapp/script.js

```diff
@@ -137,8 +148,34 @@ function visualizeMap() {
       mapTypeControlOptions: {
         mapTypeIds: ['hybrid', 'styles_map']
       }
-    });
+  });
+  
+  //Marker positioned at my house. 
+  var markerHome = new google.maps.Marker({
+    position: myHome,
+    map: map,
```

Comment:
This is perfectly fine but I wanted to note that technically you don't have to say `map: map,` here, you can simplify it to just `map,` if you'd prefer. (This is a shortcut JS allows when they're the same name.)

Reply:
Ohhh that's cool! Just made the two changes that you suggested, let me know if there is anything else I should change! :)